client/components/user-home/eventCard.js

```diff
@@ -18,14 +18,25 @@ const style = {
 const EventCard = ({type, title, details, id, notParticipant}) => (
   <div className="col s12 m6 l4 xl3">
     <div className={`card hoverable ${style[type].background}`}>
-      <Link to={`/events/${id}/controller`}>
-        <div className={`card-content ${style[type].contentText}`}>
-          <span className={`card-title ${style[type].titleText}`}>
-            {title ? title : 'Placeholder title'}
-          </span>
-          <p>{details ? details : 'Placeholder content'}</p>
-        </div>
-      </Link>
+      {type === 'done' ? (
```

Comment:
Can we just have this Link to the single event page and move the controller link there?
 
Less code too.